src/core/ext/filters/rbac/rbac_service_config_parser.cc

```diff
@@ -773,9 +772,9 @@ void RbacConfig::RbacPolicy::Rules::AuditLogger::JsonPostLoad(
 // RbacConfig::RbacPolicy::Rules
 //
 
-Rbac RbacConfig::RbacPolicy::Rules::TakeAsRbac(std::string name) {
+Rbac RbacConfig::RbacPolicy::Rules::TakeAsRbac() {
   Rbac rbac;
-  rbac.name = std::move(name);
+  rbac.name = "";
```

Comment:
Looks like a no-op?

Reply:
Updated the comment. Also removed this line. I thought the absence of it would cause msan failure but now I realize std::string has default ctor.